### DISCUSSION TITLE
Is Stately Studio open sourced?

### DISCUSSION THREAD

**Discussion by cloudcompute** [statelyai/xstate#4749] [Q&A]
I don't think so. If not, xstate has almost nothing to offer to the Open source community. There are very other good alternatives to model state machines using code and JSON.

If yes, pl. share the link. Thanks.

**Comment by davidkpiano** [comment]
Stately Studio is not open-sourced, because that is how we can currently sustain development on XState.

> If not, xstate has almost nothing to offer to the Open source community.

I'm sorry you feel that way. Feel free to use the alternatives.
